### PR TITLE
[EC3] Migrated UI items for iTwin 4.x upgrade for EC3 Widget

### PR DESCRIPTION
**Pull request by smmr-dn**
Upgraded CSS variables and replaced deprecated packages/components for EC3 widget.

### CODE REVIEW

File: common/changes/@itwin/ec3-widget-react/uyen-ec3-widget-migration_2023-08-02-20-20.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/ec3-widget-react",
+      "comment": "Migrated UI items for iTwin UI 2.x and iTwinJS 4.x upgrade.",
+      "type": "minor"
```
Comment (ben-polinsky) [inline common/changes/@itwin/ec3-widget-react/uyen-ec3-widget-migration_2023-08-02-20-20.json:6]: Bumping a peer dependency to a major version is usually considered a breaking/major change.

Comment (smmr-dn) [reply common/changes/@itwin/ec3-widget-react/uyen-ec3-widget-migration_2023-08-02-20-20.json]: I just changed it to 'major'.

Comment (Guillar1) [reply common/changes/@itwin/ec3-widget-react/uyen-ec3-widget-migration_2023-08-02-20-20.json]: Hey Ben,
We understand that a major version change is a significant step, but in this case, we believe a minor bump is appropriate for this widget, given its early stage of development.

Comment (ben-polinsky) [reply common/changes/@itwin/ec3-widget-react/uyen-ec3-widget-migration_2023-08-02-20-20.json]: Ah didn't see the widget was at version 0.2.0 - yep that makes sense. Sorry for the confusion!

**Review by Guillar1 — APPROVED** [review]

**Review by aruniverse — APPROVED** [review]

**Pull request merged**